classes/local/model/cms_types.php

```diff
@@ -45,6 +45,10 @@ class cms_types extends persistent {
      * Table name.
      */
     const TABLE = 'cms_types';
+    /** The maximum amount of bytes allowed in a file uplaod. */
+    const MAX_FILEBYTES = 1000000;
```

Comment:
Are we really only limiting to < 1MB? Shouldn't we just use the site upload limit (`$CFG->maxbytes`) or even the course upload limit (moodlecourse | maxbytes)?

Reply:
Changed